config/database.yml

```diff
@@ -17,7 +18,7 @@ default: &default
 
 development:
   <<: *default
-  database: gpp-hyrax-dev
+  database: publications
```

Comment:
Database value should be `<%= ENV.fetch("DATABASE_NAME") { "publications" } %>` and read from the `.env` first before defaulting to `publications`.